doc/neps/nep-0056-array-api-main-namespace.rst

```diff
@@ -0,0 +1,524 @@
+=============================================================
+NEP 56 — Array API standard support in NumPy's main namespace
+=============================================================
+
+:Author: Ralf Gommers <ralf.gommers@gmail.com>
+:Author: Mateusz Sokół <msokol@quansight.com>
+:Author: Nathan Goldbaum <ngoldbaum@quansight.com>
+:Status: Draft
+:Type: Standards Track
+:Created: 2023-12-19
+:Resolution: TODO mailing list link
+
+
+Abstract
+--------
+
+This NEP proposes adding full support for the 2022.12 version of the array API
+standard in NumPy's main namespace for the 2.0 release.
+
+Motivation and scope
+--------------------
+
+.. note::
+
+    The main changes proposed in this NEP were presented in the NumPy 2.0
+    Developer Meeting in April 2023 (see `here
+    <https://github.com/numpy/archive/blob/main/2.0_developer_meeting/NumPy_2.0_devmeeting_array_API_adoption.pdf>`__
+    for presentations from that meeting) and given a thumbs up there. The
```

Comment:
>  (I explicitly did not give a thumbs-up for the type promotion changes in that meeting).

I'm not sure what you mean exactly, since type promotion changes are basically NEP 50. Can you clarify?

Reply:
Not yet, but I will (a few more substantial updates to make). I never considered that a main change, but since it's going to be removed from this NEP I think we're good for this paragraph.